bin.src/ci_imsim_run.py

```diff
@@ -47,24 +63,12 @@ def addArgs(cls, parser: ArgumentParser):
 class ImsimIngestRaws(IngestRaws):
     rawLocation = os.path.join(TESTDATA_DIR, "raw")
 
-    # TODO: Remove when DM-30607 is fixed (needed for MacOS only)
-    def run(self, currentState: BuildState):
-        self.arguments.num_cores, saveCores = '1', self.arguments.num_cores
-        super().run(currentState)
-        self.arguments.num_cores = saveCores
-
```

Comment:
So how are `ImsimIngestRaws` and `ImsimDefineVisits` still used? Should they be deprecated?

Reply:
They will just use their default run methods. The Imsim classes still need to exist to define the rawLocation attribute.